cluster/images/conformance/conformance-e2e.yaml

```diff
@@ -66,6 +66,8 @@ spec:
       value: "skeleton"
     - name: E2E_PARALLEL
       value: "false"
+    - name: E2E_VERBOSITY
+      value: "-v=8"
```

Comment:
`value: "8"` please

Reply:
Scratch that. please don't use `8` here, you should use the default verbosity level (don't remember what that is ... probably `4`)

we should se the `8` only in the job (test-infra configuration)